launch/src/main.ts

```diff
@@ -5,6 +5,7 @@ import config from './config.json'
 import { blake2AsU8a } from '@polkadot/util-crypto'
 import { stringToU8a, bnToU8a, BN, u8aConcat, u8aToHex } from '@polkadot/util'
 import { decodeAddress, encodeAddress } from '@polkadot/keyring'
+import '@polkadot/api-augment'
```

Comment:
https://polkadot.js.org/docs/api/FAQ/#since-upgrading-to-the-7x-series-typescript-augmentation-is-missing